tests/marqo_test.py

```diff
@@ -202,9 +204,16 @@ def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
         """
         Creates a cloud index with the given name and settings. If settings_dict or any kwargs are provided,
         they will be used to create a unique index name by hashing the settings_dict and kwargs.
-        If settings_dict is not provided, the index name will be the given index_name + the index_suffix.
+        If settings_dict and index settings kwargs are not provided,
+        the index name will be the given index_name + the index_suffix.
 
         Performs check for status of index and waits for index to be ready before returning.
+
+        Args:
+            index_name (str): Name of index to create
+            settings_dict (dict): Dictionary of settings to use when creating index,
```

Comment:
nit: worth adding a caveat here: `Please note that settings_dict override any kwargs during index creation`